docs/get-started/quick_start.md

```diff
@@ -49,14 +55,14 @@ def add(x, y):
 
 ## Creating a compiler
 
-To compile the function, you need to create a `Compiler` by specifying the function to compile and the encryption status of its inputs:
+To compile the function, you first need to create a `Compiler` by specifying the function to compile and the encryption status of its inputs:
```

Comment:
Capitaliszation: **C**ompiler

Reply:
here, it must be "Compiler" with a C, it's code